superset/models/helpers.py

```diff
@@ -1945,7 +1958,6 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
                 inner_groupby_exprs = []
                 inner_select_exprs = []
                 for gby_name, gby_obj in groupby_series_columns.items():
-                    label = get_column_name(gby_name)
```

Comment:
bycatch - this variable was not used anywhere (apparently this function is so bloated nowadays that the linters just give up on it)